### PR TITLE
Test C/I changes

### PR DESCRIPTION
**Pull request by chiragjn**
Fails text entities on c/i

### CODE REVIEW

File: ner_v1/detectors/textual/text/text_detection.py [hunk 1]
```diff
@@ -1,3 +1,6 @@
+if True:
+    print('C/I test PR')
```
Comment (srajdev) [inline ner_v1/detectors/textual/text/text_detection.py:2]: E1601 [1;31mprint statement used[0m

File: ner_v1/detectors/textual/text/text_detection.py [hunk 2]
```diff
@@ -280,7 +283,7 @@ def _get_tokens_and_indices(txt):
         except (ValueError, IndexError):
             ner_logger.exception('Error getting original text (%s, %s)' % (matched_tokens, text))
 
-        return u' '.join(matched_tokens)
+        return u' FAIL ALL TEXT TESTS ' # .join(matched_tokens)
```
Comment (srajdev) [inline ner_v1/detectors/textual/text/text_detection.py:286]: E261 at least two spaces before inline comment

**Comment by srajdev** [comment]
:+1: No lint errors found.

**Comment by ranvijayj** [comment]
retest this please

**Comment by chiragjn** [comment]
retest this please

**Comment by vishnuvnn** [comment]
retest this please

**Comment by vishnuvnn** [comment]
retest this please

**Comment by vishnuvnn** [comment]
retest this please

**Comment by vishnuvnn** [comment]
retest this please

**Comment by vishnuvnn** [comment]
retest this please

**Comment by vishnuvnn** [comment]
retest this please

**Comment by vishnuvnn** [comment]
retest this please

**Comment by vishnuvnn** [comment]
retest this please

**Comment by vishnuvnn** [comment]
retest this please

**Comment by vishnuvnn** [comment]
retest this please

**Comment by haptik-deployment** [comment]
UNIT TESTS FAILED... Please resolve issues

**Comment by vishnuvnn** [comment]
retest this please

**Comment by haptik-deployment** [comment]
UNIT TESTS FAILED... Please resolve issues

**Pull request closed without merging**